base/sysinfo.jl

```diff
@@ -159,7 +169,7 @@ total_memory() = ccall(:uv_get_total_memory, UInt64, ())
 """
     Sys.get_process_title()
 
-Get the process title. On some systems, will always return empty string. (not exported)
+Get the process title. On some systems, will always return empty string.
```

Comment:
While you're at it, perhaps add the missing article "return empty string" -> "return an empty string"? :)